<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win_jinhuo</class>
 <widget class="QDialog" name="win_jinhuo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>进货</string>
  </property>
  <widget class="QTextEdit" name="input_win_name">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hint_input_book_name">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入欲进货书名</string>
   </property>
  </widget>
  <widget class="QLabel" name="hint_book_table">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>藏书列表</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="book_table">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>301</width>
     <height>431</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>80</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>库存</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="book_name_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hint_jin_huo_table">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>进货单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="jin_huo_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认进货</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="jin_huo_table">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>311</width>
     <height>261</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>75</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>书目ID</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>供应商ID</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>进货价</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数量</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
